tests/express-pouchdb/test.js

```diff
@@ -107,6 +107,49 @@ describe('config', function () {
       });
     });
   });
+  it('should support setting admins', function (done) {
+    var cleanUpTest = function () {
+      return fse.removeAsync(TEST_DATA + 'one-shot');
+    };
+    fse.mkdirsAsync(TEST_DATA + 'one-shot').then(function () {
+      // Creates a single usage app.
+      var oneShotExpressApp = buildApp(PouchDB.defaults({
+        prefix: TEST_DATA + 'one-shot/',
+      }), {
+        configPath: TEST_DATA + 'one-shot/config.json',
+        logPath: TEST_DATA + 'one-shot/log.txt'
+      });
```

Comment:
All of the above boilerplate is certainly not elegant... But for some reasons, setting up beforeEach and afterEach (which would be the best way to isolate all tests), makes many subsequent tests failing.